docs/chat/chat.yaml

```diff
@@ -59,47 +54,39 @@ paths:
       produces:
         - application/json
       responses:
-        200:
+        '200':
```

Comment:
do you need to cover it by ''?

Reply:
> do you need to cover it by ''?

it works with, or without them, but, as we write it everywhere without quotes, i removed them